main.cpp

```diff
@@ -209,28 +211,25 @@ void ButtonFactory::designButton()
 
 void ButtonFactory::shipButtons(std::string address, int buttonType, int numToShip)
 {
-    address = "2904 Lark St. Moraine, ND 54206";
-    buttonType = 45;
-    numToShip = 140;
+    std::cout << "Ship To:" << address << " Type:" << buttonType << " Quantity:" << numToShip;
```

Comment:
- add spaces after `:` so the output is more readable.
- add `\n` or `std::endl;` at the end so a newLine is printed at the end.  Anything printed after this will be on the same line if you **don't** add these things.
- [ ] revise